src/components/MessageInput.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="message-input">
-    <Avatar :address="address" />
+    <Avatar :address="address || ''" />
```

Comment:
I think it is better to set `''` as a default value of `address` property of Avatar component instead.

Especially, if we assume that 
![Screenshot 2020-08-17 at 21 03 46](https://user-images.githubusercontent.com/9007851/90389563-24e31200-e0cd-11ea-90e2-908321d43c1b.png)
is a valid avatar of an anonymous user.